posthog/hogql_queries/experiments/test/test_funnels_statistics.py

```diff
@@ -58,16 +60,16 @@ def run_test(stats_version, calculate_probabilities, are_results_significant, ca
 
             self.assertEqual(len(probabilities), 2)
             if stats_version == 2:
-                self.assertAlmostEqual(probabilities[0], 0.15, delta=0.1)
-                self.assertAlmostEqual(probabilities[1], 0.85, delta=0.1)
+                self.assertAlmostEqual(probabilities[0], 0.149, delta=0.1)
```

Comment:
What's the reason for keeping `delta=0.1` here? The range seems too wide (0.049 -> 0.249, i.e. 4.9% to 24.9%)